README.md

```diff
@@ -123,12 +126,33 @@ var validate = validator({
       type: 'string'
     }
   }
-}, {
+}
+var validate = validator(schema, {
   verbose: true
 })
 
 validate({hello: 100});
-console.log(validate.errors) // {field: 'data.hello', message: 'is the wrong type', value: 100, type: 'string'}
+console.log(validate.errors, null, 2)
```

Comment:
I'm guessing that `null, 2` was intended for a JSON.stringfy call and not a console.log one :)

Reply:
Nice catch, thanks :) Fixed.